lib/web_ui/lib/src/engine/keyboard_binding.dart

```diff
@@ -472,7 +472,7 @@ class KeyboardConverter {
             timeStamp: timeStamp,
             type: ui.KeyEventType.up,
             physical: physicalKey,
-            logical: logicalKey(),
+            logical: _pressingRecords[physicalKey]!,
```

Comment:
Thank you for your contribution @bleroux!

@dkwingsmt is this an acceptable approach to handling the `hardware_keyboard` physical/logical key mismatch error on web? From my understanding, the code here is written to map logical keys on web similar to how other platforms do it - however, I think only the browser emits extra "Process" key events for special characters like IME composition, emojis, etc